src/modules/taskConstructor.js

```diff
@@ -0,0 +1,8 @@
+export default class Task {
+  constructor(description, id = 0) {
+    this.description = description;
+    this.editable = false;
+    this.completed = false;
+    this.id = id;
+  }
+}
```

Comment:
- [ ] Kindly notice that there is no need for the `editable` attribute, also the `id` attribute should be the `index` attribute.

```javascript
export default class Task {
  constructor(description, index = 0) {
    this.description = description;
    this.completed = false;
    this.index = index;
  }
}
```